code/modules/mob/living/carbon/xenomorph/castes/wraith/castedatum_wraith.dm

```diff
@@ -0,0 +1,144 @@
+/datum/xeno_caste/wraith
+	caste_name = "Wraith"
+	display_name = "Wraith"
+	upgrade_name = ""
+	caste_desc = "A strange xeno that utilizes its psychic powers to move out of phase with reality."
+	caste_type_path = /mob/living/carbon/xenomorph/wraith
+	tier = XENO_TIER_TWO
+	upgrade = XENO_UPGRADE_BASETYPE
+	wound_type = "screecher" //used to match appropriate wound overlays
+
+	// *** Melee Attacks *** //
+	melee_damage = 17
+
+	// *** Tackle *** //
+	tackle_damage = 21
+
+	// *** Speed *** //
+	speed = -1
+
+	// *** Plasma *** //
+	plasma_max = 200
+	plasma_gain = 15
+
+	// *** Health *** //
+	max_health = 200
+
+	// *** Evolution *** //
+	evolution_threshold = 180
+	upgrade_threshold = 120
+
+	evolves_to = list(
+		/mob/living/carbon/xenomorph/crusher,
+		/mob/living/carbon/xenomorph/ravager,
+	)
```

Comment:
This feels more like it should evolve to shrike, thematically at least.

Otherwise this should start a whole separate 'tree' of psychically powered xenos.

Reply:
I have a telekinesis based psychic T3 planned for later called the Overmind; this is essentially a placeholder for now.